### PR TITLE
doc: update versions for 1.5.0 release

### PR DESCRIPTION
**Pull request by peknis**
Updated version files and version mentions for sdk-nrf.
Updated SES link to point to v5.34a

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -234,11 +234,11 @@
 
 .. ### Source: www.segger.com
 
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_win_x64
-.. _`SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_macos
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x86`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x86
-.. _`SEGGER Embedded Studio (Nordic Edition) - Linux x64`: https://segger.com/downloads/embedded-studio/embeddedstudio_arm_nordic_linux_x64
+.. _`SEGGER Embedded Studio (Nordic Edition) - Windows x86`: Windows x86`: https://www.segger.com/downloads/embedded-studio/EmbeddedStudio_ARM_Nordic_v534a_win_x86.zip
```
Comment (greg-fer) [inline doc/nrf/links.txt:237]: Some copy-paste error.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/links.txt (hunk 1 above)
Comment (peknis) [reply doc/nrf/links.txt:237]: Please enlight me. Cannot see the error.

Comment (greg-fer) [reply doc/nrf/links.txt:237]: `Windows x86`: Windows x86`: `

Comment (peknis) [reply doc/nrf/links.txt:237]: Oh, how blind one can be. Thanks.

**Review by tejlmand — APPROVED** [review]

File: doc/versions.json [hunk 2]
```diff
@@ -17,16 +17,16 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.4.99",
-      "nrfxlib": "1.4.99",
+      "ncs": "1.5.0",
+      "nrfxlib": "1.5.0",
       "zephyr": "2.4.99",
-      "mcuboot": "1.6.99"
+      "mcuboot": "1.7.99"
```
Comment (ru-fu) [inline doc/versions.json:23]: If mcuboot is at v1.7.99 now, this also needs to be updated in nrf/doc/mcuboot/conf.py
Actually, that should have been done during the mcuboot upmerge that put the version at 1.7.99 - @carlescufi , is that missing in some checklist for upmerges?

**Comment by ru-fu** [comment]
Can you please do an upmerge and fix the conflict? I'd like to look at the output.

**Review by ru-fu — CHANGES_REQUESTED** [review]
Issues with the branch - will try to fix.

**Review by ru-fu — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Comment by peknis** [comment]
The same compliance check failure remains after rebase. CI help referred to https://stackoverflow.com/questions/29206482/pylint-duplicate-code-false-positive/30007053#30007053, but I have no clue what to do.

**Comment by tejlmand** [comment]
> The same compliance check failure remains after rebase. CI help referred to https://stackoverflow.com/questions/29206482/pylint-duplicate-code-false-positive/30007053#30007053, but I have no clue what to do.

looking into this

**Comment by thst-nordic** [comment]
I found the problem is pylint==2.7.0.  
I have a fix here: https://github.com/nrfconnect/sdk-nrf/pull/4025

**Review by thst-nordic — APPROVED** [review]

**Comment by peknis** [comment]
SHA updated to west.yml.

**Pull request merged**